risc0/zkvm/src/host/protos/api.proto

```diff
@@ -206,6 +206,7 @@ message SessionInfo {
   uint32 segments = 1;
   bytes journal = 2;
   base.ExitCode exit_code = 3;
+  Asset receipt_claim = 4;
```

Comment:
Why is this an Asset?

Reply:
It could be bytes instead. I just used Asset because that's how we package the other opaque messages.

Reply:
Why is it opaque?

Reply:
Ah, because the `ReceiptClaim` type is in `core.proto`